JetNews/app/src/main/java/com/example/jetnews/ui/theme/Color.kt

```diff
@@ -23,3 +23,6 @@ val Red300 = Color(0xffea6d7e)
 val Red700 = Color(0xffdd0d3c)
 val Red800 = Color(0xffd00036)
 val Red900 = Color(0xffc20029)
+
+val ArticleIconBackground = Color(0xff073042)
+val ArticleIconColor = Color(0xff32dd84)
```

Comment:
You're not using these variables anymore right?

Reply:
Just removed them